tech/languages/csharp/dotnet-installation.md

```diff
@@ -17,21 +17,29 @@ $ sudo dnf copr enable @dotnet-sig/dotnet
 
 ### .NET Core SDK
 
-.NET Core SDK will enable building and publishing C# source code.
+.NET Core SDK enables building and publishing of C# source code.
+
+To install the latest SDK version:
+
+```
+$ sudo dnf install dotnet
+```
+
+To install the latest SDK for a specific .NET Core Runtime, where the _x_ stands for major, _y_ for minor version of the runtime:
 ```
-$ sudo dnf install dotnet-sdk-2.0
+$ sudo dnf install dotnet-sdk-x.y
 ```
 
 ### .NET Core Runtime only
 
-You can install runtime only, if you are just deploying already prebuilt application.
+To install runtime only, for example to merely deploy already prebuilt applications, where _x_ stands for major and _y_ stands for minor version:
```

Comment:
Could you maybe just put here the command to search for the latest version. Or isn't it really possible to do:

`$ sudo dnf install dotnet-sdk`  

?

Reply:
That is equal to just `dotnet`.
There are three use-cases that we are trying to cover:
dnf install dotnet == dotnet-sdk - latest sdk with latest framework
dnf install dotnet-sdk-version - latest sdk with specific framework version
dnf install dotnet-runtime-version - specific framework version, without any sdk

Reply:
So, the way I understand it, you just need `dnf install dotnet-runtime-version` (and without version it does not work, right?). So I'd drop the rest.

Reply:
It would seem that my comment decided to disappear in the void. TLDR of what's gone: `dotnet-runtime-version` is not the same as `dotnet-sdk-version`, or `dotnet` and there are use-cases for all. Some more info here: https://docs.microsoft.com/en-us/dotnet/core/build/distribution-packaging